cli/src/utils.js

```diff
@@ -93,8 +94,61 @@ async function runCommand(cmd, parameters = [], options = {}) {
   });
 }
 
+async function findCRI() {
+  // Resolve the CRI to be used
+  if (await which('docker-compose', { nothrow: true }))
+    return ['docker-compose']
+
+  if (await which('docker', { nothrow: true }))
+    return ['docker', 'compose']
+
+  if (await which('podman-compose', { nothrow: true }))
+    return ['podman-compose']
+
+  if (await which('podman', { nothrow: true }))
+    return ['podman', 'compose']
+
+  throw new Error('Cannot find a valid runtime to run containers');
+}
+
+const runComposeActions = {
```

Comment:
Wdyt about merging 'docker'/'docker-compose' and 'podman'/'podman-compose'. The compose actions are the same. Do you think they might diverge in the future?

Reply:
Yes, I think it might.
Actually, docker-compose is deprecated and won't be updated anymore. I've added it for compatibility reasons with people who still have an old version of docker. Docker compose is a go rewrite of docker-compose and it will continue to live apart from it.
For podman/podman-compose i doubt it will differ for now. Podman compose is just a wrapper around docker-compose or podman-compose